blueocean-dashboard/src/main/js/creation/github/steps/GithubCredentialStep.jsx

```diff
@@ -45,18 +82,61 @@ export default class GithubCredentialsStep extends React.Component {
         return null;
     }
 
+    _getInstructions(stateId) {
```

Comment:
I will probably refactor this method into a composed component regardless of whether I do a more extensive refactor on this step.